test/functional/test_runner.py

```diff
@@ -222,6 +222,7 @@
     'rpc_help.py',
     'feature_help.py',
     'feature_shutdown.py',
+    'framework_test.py',
```

Comment:
Not sure if it makes sense to add an unscoped general "framework test". In the past we either assumed that the test framework is RIGHT(TM)(C) or added an inline unit test in the file that needed that part of the test framwork. For example,

```
$ git grep 'This is a sanity check of our testing fram'
test/functional/p2p_segwit.py:        # This is a sanity check of our testing framework.

Reply:
If you want to keep the file, it could make sense to rename it to `framework_test_bn2vch` or so.

Reply:
Yeah, an inline test would be fine, but it's not clear where. This code is indirectly used from a number of locations I expect.

Re separate file: That's perhaps better; what about `framework_test_script`?

Reply:
Done.